app/controllers/projects_controller.rb

```diff
@@ -9,13 +9,12 @@ class ProjectsController < ApplicationController
   # GET /projects/new
   def new
     @project = Project.new
-    @project_image = ProjectImage.new
   end
 
   # GET /projects
   # GET /projects.json
   def index
-    @projects = Project.all
+    @projects = Project.all.select { |project| !project.attributes.hidden }
```

Comment:
The feature I intended was to hide at all those projects. They would be
visible just from the "My Projects" page. It is a intended difference. Have
faith :)

Em qui, 26 de fev de 2015 às 13:22, Diego de Araújo Martinez Camarinha <
notifications@github.com> escreveu:

> In app/controllers/projects_controller.rb
> <https://github.com/mezuro/prezento/pull/183#discussion_r25440163>:
>
> >    end
> >
> >    # GET /projects
> >    # GET /projects.json
> >    def index
> > -    @projects = Project.all
> > +    @projects = Project.all.select { |project| !project.attributes.hidden }
>
> Something may have been lost. Look at the latest commit made by @marcheing
> <https://github.com/marcheing>. The action index is a bit different.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mezuro/prezento/pull/183/files#r25440163>.
>